### DISCUSSION TITLE
Lost bluetooth connection

### DISCUSSION THREAD

**Discussion by Fernandroide** [syssi/esphome-jk-bms#618] [Q&A]
When I install or update bms-jk.yaml I receive data from bms, but if I restart the esp32 (for example if power fails) the bluetooth connection is lost and does not reconnect unless you reinstall or upgrade the yaml in ESP32

**Comment by syssi** [comment]
Are you sure? The BMS is able to handle a single connection only. If the connection gets lost you have to wait a few seconds/minutes before the BMS is able to pick up the next connection/a new connection.

If this doesn't help please provide a log which indicates the ESP isn't able to connect.

**Reply by Fernandroide** [reply]
Hello Syssi,
first of all, thank you for answering. Sorry I didn’t answer you before but the installation is in a motorhome and there was no internet coverage
Already working well, when you start the question about my problem suspected it was a problem with my configuration because I had not found any comment with the same problem. So I restarted the esp32, download the yaml example and modified it for my BMS and everything worked perfectly.
What I do not know is that there was a problem in the previous configuration, because it worked well when I installed it, but if the esp32 lost power did not establish bluetooth connection
